samples/sampler/stories/util/icons.js

```diff
@@ -0,0 +1,141 @@
+// Icons For Samples
```

Comment:
please rename the folder to "helper" , just like in Sansdtone. We need consistency
and fix the imports in the stories afterwards